fbw-a32nx/src/base/flybywire-aircraft-a320-neo/SimObjects/AirPlanes/FlyByWire_A320_NEO/aircraft.cfg

```diff
@@ -3,15 +3,15 @@ major = 1
 minor = 0
 
 [GENERAL]
-atc_type = "TT:ATCCOM.ATC_NAME AIRBUS.0.text"
-atc_model = "TT:ATCCOM.AC_MODEL_A20N.0.text"
+atc_type = "Airbus"
+atc_model = "FBWA32NX"
 Category = "airplane"
 performance = "Engines\nCFM-56 5B rated at 26,500 lbs\n\nTypical Cruise Speed\n0.78 Mach at 35,000 ft\n\nMax Speed\nVmo 350 kts    Mmo 0.82 Mach\n\nRange(150 pax)\n2,900 nm (5,374 km)\n\nMax Fuel\n42,230 lbs (6,303 US gal)\n\nMax Takeoff Weight\n162,040 lbs\n\nMax Gross Weight (taxi weight)\n162,900 lbs\n\nMax Payload\n42,370 lbs"
 editable = 1
 wip_indicator = 2
 icao_type_designator = "A20N"
 icao_manufacturer = "AIRBUS"
-icao_model = "A-320neo"
+icao_model = "A32NX"
```

Comment:
This was correct before https://www.icao.int/publications/doc8643/pages/search.aspx